components/features/Dashboard/Applications/InteractiveQuiz/Assignment/Assignment.test.tsx

```diff
@@ -3,26 +3,29 @@ import { render } from "@testing-library/react";
 import Assignment from "./Assignment";
 import { RouterContext } from "next/dist/shared/lib/router-context";
 import { createMockRouter } from "../../../../../../mocks/createMockRouter";
+import { AuthProvider } from "../../../../../../context/Authentication/";
 
 describe("Assignment component test suite", () => {
   test("testing render component", async () => {
     render(
       <RouterContext.Provider value={createMockRouter({})}>
-        <Assignment
-          setCurrentActiveAssignment={() => {}}
-          assignment={{
-            id: "id",
-            type: "assignment",
-            name: "Your assignment",
-            description: "do this assessment",
-            additional_info: {},
-            released_time: "22:00:00",
-            end_working_time: "22:00:00",
-          }}
-        />
+        <AuthProvider>
+          <Assignment
+            setCurrentActiveAssignment={() => {}}
+            assignment={{
```

Comment:
Hello, Rashad. just to make sure this placeholder will be replaced with the actual data later, right?

Reply:
Forgive me, I'm not aware of the project structure :)